### DISCUSSION TITLE
[Feature] Basic image editing

### DISCUSSION THREAD

**Discussion by alberto8051** [immich-app/immich#1991] [Feature Request]
### The feature

Hi Everyone

I have some suggestion to improve this great software. I'll open a Feature Request for each one. 
First at all, I think it would be useful some basic edition on the images like rotate and edit EXIF data in order to correct the date of the image.

Thank you 
Best Regards
Alberto

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by axgd-code** [comment]
I fully agree with the proposal to include photo editing functions. Among these, cropping and rotation appear to be top-priority features, offering essential flexibility during image editing.

Furthermore, it would be beneficial to implement a confirmation prompt after each modification, asking the user whether they want to create a modified copy or apply changes directly to the original. This ensures a more personalized user experience and helps prevent accidental modifications to the original image.

These enhancements would undoubtedly make the tool more user-friendly 👍

**Comment by Dschogo** [comment]
#3271 should include all that (also #5151 for rotating)

**Comment by Skivling** [comment]
I +1 this, recently took a couple hundred landscape photos and they all appear the wrong rotation.

**Comment by Murguth** [comment]
And a bit of filters and color correction. :P

**Comment by bo0tzz** [comment]
#24155
